Framework/Algorithms/src/CalculatePolynomialBackground.cpp

```diff
@@ -199,38 +207,61 @@ executeFit(Mantid::API::Algorithm &fit, const std::string &function,
  *  @param parameters a vector containing the polynomial coefficients
  *  @return a function string
  */
-std::string makeFunctionString(const std::vector<double> &parameters) {
+std::string makeFunctionString(const std::string &name,
+                               const std::vector<double> &parameters) {
   const auto degree = parameters.size() - 1;
```

Comment:
Is there any guarantee somewhere that parameters size is at least 1?

Reply:
Yes, in exec the size of parameters is degree + 1 and there is validation to check degree is > 0